imagersite/imager_images/models.py

```diff
@@ -0,0 +1,75 @@
+"""Model photos and albums for database items."""
+
+from django.db import models
+from django.conf import settings
+from django.utils.encoding import python_2_unicode_compatible
+
+# Create your models here.
+
+PUBLISHED_CHOICES = [('private', 'private'), ('shared', 'shared'), ('public', 'public')]
+PUBLISHED_DEFAULT = PUBLISHED_CHOICES[0]
+
+
+@python_2_unicode_compatible
+class Photo(models.Model):
+    """Model a single photo in database."""
+
+    owner = models.ForeignKey(settings.AUTH_USER_MODEL, related_name='photos')
+    albums = models.ManyToManyField('Album', related_name='photos')
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    date_uploaded = models.DateTimeField(auto_now_add=True)
+    date_modified = models.DateTimeField(auto_now=True)
+    date_published = models.DateTimeField(auto_now_add=True)
```

Comment:
How would you expect the datestamp to function when a user unpublishes an album, and then republishes it?